src/main/java/org/threadly/litesockets/tcp/SSLClient.java

```diff
@@ -338,8 +316,8 @@ public void writeForce(ByteBuffer buffer) {
               super.writeForce(tmpBB);
             }
           } catch(Exception e) {
-            if(!handshakeFuture.isDone()) {
-              handshakeFuture.setFailure(e);
+            if(! handshakeFuture.isDone() && 
```

Comment:
Yeah, that was how I had it before, and when I merged in your code it changed to that, I actually liked it because we save a few checks if it was already done.  Its an extremely minor optimization i know, if you think its worth making the code a little clear we can move it back to a single if(set.